aie_runtime_lib/AIE/aiesim/Makefile

```diff
@@ -23,6 +23,9 @@ endif
 ifeq ($(XILINX_VITIS_AIETOOLS),)
 XILINX_VITIS_AIETOOLS = $(dir $(shell which aiesimulator))/..
 endif
+
+MLIR_AIE_PROJ = $(notdir $(patsubst %/,%,$(dir $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST))))))))
```

Comment:
The kind of line requiring an English summary about what it does. :-)

Reply:
Thanks for the suggestion @keryell. I'll add it for my next push.